clustered/client/src/test/java/org/ehcache/clustered/client/internal/service/DefaultClusteringServiceTest.java

```diff
@@ -543,7 +543,7 @@ public void testGetServerStoreProxySharedAutoCreate() throws Exception {
         getSharedStoreConfig(targetPool, serializationProvider, Long.class, String.class);
 
     ServerStoreProxy serverStoreProxy = service.getServerStoreProxy(
-        getClusteredCacheIdentifier(service, cacheAlias), storeConfiguration, Consistency.EVENTUAL);
+        getClusteredCacheIdentifier(service, cacheAlias), storeConfiguration, Consistency.EVENTUAL, null);
```

Comment:
The default method for here as well. But don't you want to test if the callback is called as expected?

Reply:
Seems a little out of this tests remit to me.  I'd have to wire an awful lot more stuff to test this properly

Reply:
Which means the callback isn't tested?